app/graphql/schema/queries/all-customer-query-tickets.js

```diff
@@ -0,0 +1,43 @@
+const cosmos = require('../../../cosmos')
+const { cosmosConfig } = require('../../../config')
+const { convertCosmosTimestamp } = require('../../../utils')
+
+const allCustomerQueryTickets = async (_root, args, context) => {
+  try {
+    const { queriesDatabase } = await cosmos()
+
+    const querySpec = {
+      query: 'SELECT * FROM customerQueryResponse cq ORDER BY cq._ts ASC'
+    }
+
+    const response = await queriesDatabase
+      .container(cosmosConfig.queriesContainer)
+      .items.query(querySpec)
+      .fetchAll()
+
+    return {
+      status: {
+        code: 200,
```

Comment:
As below, these are hard coded

Reply:
Same as previous comment, this has been removed now.